modules/timer/timer.cpp

```diff
@@ -0,0 +1,327 @@
+//=====[Libraries]=============================================================
+#include "timer.h"
+#include "FSM_principal.h"
+#include "UART.h"
+
+//=====[Declaration and initialization of private global variables]============
+
+// The following arrays contains four values, one for each seven-segment display
+// The seven segment display is not implemented yet
+static  int     exerciseModeInitialValues   [4]     = {0,0,1,0}; // Initialization with random value
```

Comment:
No queda claro dos cosas:
1) A qué te referís con valor random si acá hay un valor concreto, que definitivamente no es random
2) A qué te referís con "The seven segment display is not implemented yet", ¿cuándo se implementaría?

Reply:
Claro, puse 'aleatorio' porque era un valor arbitrario que lo puse nomás para probar el funcionamiento sin esperar mucho tiempo a que termine la cuenta regresiva. Lo voy a cambiar por 'valor inicial'.

Reply:
Ok, de acuerdo, muchas gracias por tus aclaraciones.